ocb3/tests/kats.rs

```diff
@@ -0,0 +1,250 @@
+#![allow(non_snake_case)]
+
+use aead::{
+    consts::{U12, U8},
+    AeadInPlace, KeyInit,
+};
+use aes::{Aes128, Aes192, Aes256, Block};
+use hex_literal::hex;
+use ocb3::{Aes128Ocb3, Aes256Ocb3, AesOcb3, GenericArray};
+
+/// Test vectors from https://www.rfc-editor.org/rfc/rfc7253.html#appendix-A
+#[test]
+fn rfc7253_sample_results() {
+    let key = GenericArray::from(hex!("000102030405060708090A0B0C0D0E0F"));
+    struct Kat {
+        nonce: Vec<u8>,
+        associated_data: Vec<u8>,
+        plaintext: Vec<u8>,
+        ciphertext: Vec<u8>,
+    }
+    let kats = [
+            Kat {
+                nonce: hex!("BBAA99887766554433221100").to_vec(),
+                associated_data: hex!("").to_vec(),
+                plaintext: hex!("").to_vec(),
+                ciphertext: hex!("785407BFFFC8AD9EDCC5520AC9111EE6").to_vec(),
+            },
```

Comment:
These tests are better implemented using the `new_test` macro. If you want, I can generate blobby file for these tests a bit later.